src/main/java/de/refactoringbot/refactoring/RefactoringHelper.java

```diff
@@ -4,333 +4,289 @@
 import java.io.FileNotFoundException;
 import java.nio.file.InvalidPathException;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Optional;
+import java.util.Set;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.github.javaparser.JavaParser;
+import com.github.javaparser.Position;
 import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.Node;
 import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
 import com.github.javaparser.ast.body.FieldDeclaration;
 import com.github.javaparser.ast.body.MethodDeclaration;
-import com.github.javaparser.ast.expr.MethodCallExpr;
 import com.github.javaparser.printer.lexicalpreservation.LexicalPreservingPrinter;
 import com.github.javaparser.resolution.UnsolvedSymbolException;
 import com.github.javaparser.resolution.declarations.ResolvedMethodDeclaration;
 import com.github.javaparser.resolution.declarations.ResolvedReferenceTypeDeclaration;
 import com.github.javaparser.resolution.types.ResolvedReferenceType;
 
 import de.refactoringbot.model.exceptions.BotRefactoringException;
-import de.refactoringbot.model.javaparser.ParserRefactoring;
 
 /**
- * This class contains many methods that can be used by multiple
- * Refactoring-Classes.
- * 
- * @author Stefan Basaric
- *
+ * Utility methods for use in performing refactorings
  */
 public class RefactoringHelper {
 
-	private static final Logger logger = LoggerFactory.getLogger(RefactoringPicker.class);
+	private static final Logger logger = LoggerFactory.getLogger(RefactoringHelper.class);
 
 	private RefactoringHelper() {
 	}
 
 	/**
-	 * This method scans all Java files for methods that match the passed method
-	 * signature and adds them to the given refactoring
-	 * 
-	 * @param refactoring
-	 * @param allJavaFiles
-	 * @return
-	 * @throws FileNotFoundException
+	 * @param classOrInterface
+	 * @param methodSignature
+	 * @return true if local method signature is present in given class or
+	 *         interface, false otherwise
 	 */
-	public static ParserRefactoring findAndAddMethods(ParserRefactoring refactoring, List<String> allJavaFiles,
-			String methodSignature) throws FileNotFoundException {
-
-		// Iterate all Java-Files
-		for (String javaFile : allJavaFiles) {
-			// parse a file
-			FileInputStream filepath = new FileInputStream(javaFile);
-			CompilationUnit compilationUnit = LexicalPreservingPrinter.setup(JavaParser.parse(filepath));
-
-			// Get all Classes
-			List<ClassOrInterfaceDeclaration> classes = compilationUnit.findAll(ClassOrInterfaceDeclaration.class);
-			// Iterate all Classes
-			for (ClassOrInterfaceDeclaration currentClass : classes) {
+	public static boolean isLocalMethodSignaturePresentInClassOrInterface(ClassOrInterfaceDeclaration classOrInterface,
```

Comment:
Method name contains LocalMethodSignature, but param is called methodSignature. I personally think, it is enough to call the method something like "classOrInterfaceContainsMethod" or "isMethodInClassOrInterface"